packages/nextjs/src/components/BYOCWrapper.tsx

```diff
@@ -1,25 +1,6 @@
-﻿import {
-  BYOCWrapper,
-  BYOCComponentParams,
-  fetchBYOCComponentServerProps,
-} from '@sitecore-content-sdk/react';
-import { GetComponentServerProps } from '../sharedTypes/component-props';
+﻿import { BYOCWrapper } from '@sitecore-content-sdk/react';
 
 /**
- * TODO: remove when framework agnostic forms implemented
- * This is a repackaged version of the React BYOCWrapper component with support for
- * server rendering in Next.js (using component-level data-fetching feature of Content SDK).
+ * This is a repackaged version of the React BYOCWrapper component for Next.js.
  */
-
-/**
- * Will be called during SSG or SSR
- * @param {ComponentRendering} rendering
- * @returns {GetStaticPropsContext | GetStaticPropsContext} context with type depending on SSR or SSG mode
- */
-export const getComponentServerProps: GetComponentServerProps = async (rendering) => {
```

Comment:
this is a breaking change that will affect BYOC (and FEAAS, for the FEAAS counterpart) in pages router. This function should be left as is, until getComponentServerProps is deprecated or removed completely